### ISSUE TITLE
Thumbs display briefly, and then whole page is black

### ISSUE THREAD

**Issue by isaacullah** [ramswaroop/photography#17]
Strangely, your preview images render fine, but when I add my own images, I get a black main page. I can click each square where the thumbs are supposed to be, and the light table pops up with the correct image. When I refresh or load the page, I can see the thumbs loading for half a second, and then it goes back to all black. The header and footer are there. I used your gulp script to resize the thumbs, so I can't figure out what's going wrong.
Here's a link to my site so you can see what it looks like: https://isaacullah.github.io/photography/

**Comment by ramswaroop** [comment]
Hi, I checked your repo and you've modified the `index.html` file incorrectly. Please compare this file with mine and make required corrections, it should work after that.

**Comment by isaacullah** [comment]
Thanks so much for taking a look, and also for making this template available. I made the comparison, and merged all changes that I could find, but still no luck. To demonstrate, I also put a totally unchanged version of your index.html into my gh-pages branch as "index2.html", and it still does not work:
My very slightly modified index.html: https://isaacullah.github.io/photography/index.html
Your index.html: https://isaacullah.github.io/photography/index2.html

The only changes I made to my index.html were some of the social tags (e.g., adding flickr instead of linkedin), and I merged most of them back to the way it was. Was there some other mistake I made that you saw that I am perhaps missing? I used a merge tool, but perhaps I am being very dense about this...

Thanks again!!
